crates/bevy_render/src/render_resource/shader.rs

```diff
@@ -259,30 +259,24 @@ impl AssetLoader for ShaderLoader {
     ) -> BoxedFuture<'a, Result<Shader, Self::Error>> {
         Box::pin(async move {
             let ext = load_context.path().extension().unwrap().to_str().unwrap();
-
+            let path = load_context.asset_path().to_string();
+            // On windows, the path will inconsistently use \ or /.
+            // TODO: remove this once AssetPath forces cross-platform "slash" consistency
```

Comment:
We already have an issue open for that, or is it tracked inside any existing issue?

I always find a good idea linking also the issue on TODO, to keep track when the TODO should be removed.

Reply:
Good call! Created the issue and added a reference.